### PR TITLE
fix: Required notes should be read for form fields, acceptable file types should be read for file upload

### PR DESCRIPTION
**Pull request by haworku**
## Summary
- Required notes should be read in relation to form fields
- State user knows accepted file formats on doc upload

#### Related issues
https://qmacbis.atlassian.net/browse/OY2-13570
https://qmacbis.atlassian.net/browse/OY2-13574

#### Test cases covered
- displays correct form guidance - on SubmissionType, ContractDetails, RateDetails pages
- displays correct form guidance for contract only submission and displays correct form guidance for contract and rates submission - on Contacts page

## QA guidance
- Make sure required fields are announced as required by screenreader (voiceover and JAWS)

### CODE REVIEW

File: services/app-web/src/pages/StateSubmission/Contacts/Contacts.tsx [hunk 1]
```diff
@@ -189,273 +189,661 @@ export const Contacts = ({
...
-            newStateContactNameRef.current = null
-        }
-        if (focusNewActuaryContact) {
-            newActuaryContactNameRef.current &&
-                newActuaryContactNameRef.current.focus()
-            setFocusNewActuaryContact(false)
-            newActuaryContactNameRef.current = null
-        }
-    }, [focusNewContact, focusNewActuaryContact])
+React.useEffect(() => {
```
Comment (haworku) [inline services/app-web/src/pages/StateSubmission/Contacts/Contacts.tsx:198]: It seems like prettier was never run on this file ? Or something ... sorry for the wild diff here

**Comment by haworku** [comment]
The cypress failures are cypress not picking up the programs combobox anymore and I don't know why. Trying to troubleshoot

**Comment by haworku** [comment]
This PR is ready to be merged once it code reviewed.

File: services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.tsx [hunk 2]
```diff
@@ -462,6 +474,7 @@ export const RateDetails = ({
                                                         disabled: false,
                                                         id: 'rateDateEnd',
                                                         name: 'rateDateEnd',
+                                                        'aria-required': true,
```
Comment (macrael) [inline services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.tsx:477]: How come these are = true instead of just aria-required?

**Review by macrael — APPROVED** [review]
Awesome. Happy new year!

**Pull request merged**